### DISCUSSION TITLE
Auto-detect system language

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#25607] [Ideas]
Automatically detect system language and change osu! language to match a system language even tho it's not fully translated yet, Like what (mostly) all apps and games do (idk this could be a bad idea)

**Comment by peppy** [comment]
The framework already prefers the system localisation. What language/culture is your system set to?

**Reply by ghost** [reply]
Oh I'm actually not aware of this feature, Thank you for clarifying this to me

**Reply by ghost** [reply]
And yep I can confirm that this feature work properly on my new phone but not my old phone... Weird. Also please bring Vulkan support to android please
